docs/app/pages/components/components-sections/input-controls/tags/tags.component.html

```diff
@@ -106,15 +109,15 @@
 </div>
 
 <ng-template #demoTagTemplate let-tag="tag" let-index="index" let-api="api">
-    <span class="m-r-sm"><i class="hpe-icon hpe-tag"></i></span>
+    <i class="ux-icon ux-icon-tag tag-padding" *ngIf="addTagIcon"></i>
```

Comment:
Perhaps we could just target the `ux-icon-tag` class to add the styling rather than creating an additional class

Reply:
updated to use <code>ux-icon-tag</code>